core/src/main/scala/org/apache/spark/security/CryptoStreamUtils.scala

```diff
@@ -102,4 +150,34 @@ private[spark] object CryptoStreamUtils extends Logging {
     }
     iv
   }
+
+  /**
+   * This class is a workaround for CRYPTO-125, that forces all bytes to be written to the
+   * underlying channel. Since the callers of this API are using blocking I/O, there are no
+   * concerns with regards to CPU usage here.
```

Comment:
No. As the comment states, it's a workaround for a bug in the commons-crypto library, which would affect the code being added.